### DISCUSSION TITLE
Find specific data in array of object in API search

### DISCUSSION THREAD

**Discussion by SavArmadillo** [directus/directus#14082] [Archive]
Hello!

I've been trying to minimize my front operations lately so I tried to do some specific filters in the api calls.

I have these following objects in my database : 
```json
{
  "id": "X",
  "person" : {
    "characteristics" : [
      {
        "colors" : [
          "green",
          "lime-green",
          ...
        ]
      }
    ]
  }
}
```

I want to add that ***characteristics*** is a json value with a repeater interface

Is there a way to use the filter function of the api to get all of the objects that contains the **green** value?

**Comment by rijkvanzanten** [comment]
There's currently no way to search/filter _in_ a JSON value. Given an upvote to https://github.com/directus/directus/discussions/7277 to voice your support for such a feature 🙂

**Reply by SavArmadillo** [reply]
Alright, will do!

thanks
